test/ingress/conformance/tests/httproute-rewrite.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    higress.io/rewrite-target: "/foo"
+  name: rewrite-annotation
+  namespace: higress-conformance-infra
+spec:
+  rules:
+  - http:
+      paths:
+      - pathType: Regex
```

Comment:
Error:      	Received unexpected error:
        	            	Ingress.extensions "rewrite-annotation" is invalid: spec.rules[0].http.paths[0].pathType: Unsupported value: "Regex": supported values: "Exact", "ImplementationSpecific", "Prefix"